gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -66,7 +67,12 @@ export const wrapCommand = (command) => {
 
       const latestSlot = await this.provider.connection.getSlot()
       const recentBlock = await this.provider.connection.getBlock(latestSlot)
-      const tx = makeTx(rawTxs, {
+      if (!!recentBlock == false) {
+        return {
+          responses: [],
+        }
+      }
```

Comment:
This does not give the user too much information, let's throw instead:
```suggestion
      if (!recentBlock) {
        throw new Error('Block not found. Could not generate message data')
      }
```